client/src/app/components/nova/address/section/nft/Nft.tsx

```diff
@@ -56,6 +63,10 @@ const Nft: React.FC<NftProps> = ({ nftOutput }) => {
             if (metadata) {
```

Comment:
Will this hook "recompute" after `metadata` is set (on L60) ? I would say maybe not as metadata is not in the dependencies 🤔

Reply:
Ah, i think it was supposed to be `if (nftIssuerId) {...`